src/authorization/exception/user.exception.ts

```diff
@@ -1,7 +1,14 @@
+import { BadRequestException } from '@nestjs/common';
```

Comment:
Should this exception be thrown globally whenever an inactive user tries to perform operations?